.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-json
+    -   id: check-yaml
+    -   id: check-merge-conflict
+    -   id: check-added-large-files
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+    -   id: black
+-   repo: https://github.com/pycqa/flake8
+    rev: '3.9.0'
+    hooks:
+    -   id: flake8
+        args:
+        - "--max-line-length=88"
+        # these are errors that will be ignored by flake8
+        # check out their meaning here
+        # https://flake8.pycqa.org/en/latest/user/error-codes.html
+        - "--ignore=E501,E203,W503"
```

Comment:
Do we want to ignore E501 (max line length)? I'm okay with this, just making sure there is consensus. E503 also looks useful